project/driver_controls/inc/driver_device.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
Hmm, I'm not sure if `driver_device` is the most straightforward module name. I see that it seems to be more an IO management module. Maybe something like `driver_io` might be easier to understand?